.swiftlint.yml

```diff
@@ -56,6 +56,9 @@ function_body_length:
   warning: 100
   error: 150
 
+nesting:
+  type_level: 2
+
```

Comment:
This was necessary for the events when they were in the project itself as the events are like below, but maybe we should leave this in anyway?

```
struct AnalyticsEvents {
    struct SomethingToTrack {
        enum PropertyValues {
            ...
        }
    }
}
```